app/controllers/api/base_controller.rb

```diff
@@ -39,8 +39,8 @@ class BaseController < ActionController::Base
 
     rescue_from Foreman::MaintenanceException, :with => :service_unavailable
 
-    def get_resource
-      instance_variable_get(:"@#{resource_name}") || raise('no resource loaded')
+    def get_resource(options = {})
```

Comment:
Maybe it's time to start using keyword arguments?
@dLobatog, what do you think?